src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        String[] mashine = new String[] {"Truck", "Bulldozer", "Excavator"};
```

Comment:
The array `mashine` is declared but never used in the code. If this array is not needed, it should be removed to keep the code clean and avoid unnecessary declarations.